pages/customers/edit/[id].js

```diff
@@ -33,47 +38,49 @@ export default function EditCustomer() {
   return (
     <>
       <Header />
-      <Form onSubmit={(event) => handleEditSubmit(event, id)}>
-        <label htmlFor="name">Kundenname:</label>
-        <Input
-          id="name"
-          defaultValue={customer ? customer.name : ""}
-          name="name"
-          type="text"
-          required
-        />
-        <label htmlFor="street">Straße und Hausnummer:</label>
-        <Input
-          defaultValue={customer ? customer.street : ""}
-          id="street"
-          name="street"
-          type="text"
-          required
-        />
-        <label htmlFor="location">Ort:</label>
-        <Input
-          id="location"
-          defaultValue={customer ? customer.location : ""}
-          name="location"
-          type="text"
-          required
-        />
-        <label htmlFor="areaCode">Postleitzahl:</label>
-        <Input
-          defaultValue={customer ? customer.areaCode : ""}
-          id="areaCode"
-          name="areaCode"
-          type="text"
-          minLength="5"
-          maxLength="5"
-          pattern="[0-9]{5}"
-          required
-        />
-        <div style={{ height: 20 }} />
-        <Button size="m" type="submit">
-          Bestätigen
-        </Button>
-      </Form>
+      <FormContainer>
+        <Form onSubmit={(event) => handleEditSubmit(event, id)}>
+          <label htmlFor="name">Kundenname:</label>
+          <Input
+            id="name"
+            defaultValue={customer ? customer.name : ""}
+            name="name"
+            type="text"
+            required
+          />
+          <label htmlFor="street">Straße und Hausnummer:</label>
+          <Input
+            defaultValue={customer ? customer.street : ""}
+            id="street"
+            name="street"
+            type="text"
+            required
+          />
+          <label htmlFor="location">Ort:</label>
+          <Input
+            id="location"
+            defaultValue={customer ? customer.location : ""}
+            name="location"
+            type="text"
+            required
+          />
+          <label htmlFor="areaCode">Postleitzahl:</label>
+          <Input
+            defaultValue={customer ? customer.areaCode : ""}
+            id="areaCode"
+            name="areaCode"
+            type="text"
+            minLength="5"
+            maxLength="5"
+            pattern="[0-9]{5}"
+            required
+          />
+          <div style={{ height: 20 }} />
+          <Button size="m" type="submit">
+            Bestätigen
+          </Button>
+        </Form>
+      </FormContainer>
```

Comment:
Since you have nearly the same form in your customers/add.js:
Could it be a good idea to make this form a component and use it anywhere you want?

Reply:
Thats true!